### PR TITLE
Bump current app versions

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR bumps the current app versions from 2.4 to 2.4.1.

iOS release: https://github.com/corona-warn-app/cwa-app-ios/releases/tag/v2.4.1
Android release: https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.4.1

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@Ein-Tim 
It looks like 2.4.2 is on its way for 
Android release: https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.4.2

File: gulpfile.js [hunk 1]
```diff
@@ -354,10 +354,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.6'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.4'))
+    .pipe(replace('[ios.current-app-version]', '2.4.1'))
     .pipe(replace('[android.latest-os-version]', '11'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.4'))
+    .pipe(replace('[android.current-app-version]', '2.4.1'))
```
Comment (MikeMcC399) [inline gulpfile.js:360]:
```suggestion
    .pipe(replace('[android.current-app-version]', '2.4.2'))
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Android 2.4.2 has been released on GitHub.

**Comment by Ein-Tim** [comment]
Thanks @MikeMcC399! I commited your suggested change.

File: gulpfile.js (hunk 1 above)
Comment (Ein-Tim) [reply gulpfile.js]: Done!

**Review by heinezen — APPROVED** [review]
Thx

**Pull request merged**